src/app-layout/__tests__/mobile.test.tsx

```diff
@@ -50,16 +51,23 @@ function AppLayoutWithControlledNavigation({
   );
 }
 
-describeEachAppLayout({ themes: ['classic', 'refresh'], sizes: ['mobile'] }, ({ theme }) => {
+describeEachAppLayout({ sizes: ['mobile'] }, ({ theme }) => {
   // In refactored Visual Refresh different styles are used compared to Classic
-  const mobileBarClassName = theme === 'refresh' ? testUtilsStyles['mobile-bar'] : toolbarStyles['mobile-bar'];
-  const drawerBarClassName =
+  const mobileBarClassName =
     theme === 'refresh'
-      ? visualRefreshRefactoredStyles['drawers-mobile-triggers-container']
-      : drawersMobileStyles['drawers-container'];
-  const blockBodyScrollClassName =
-    theme === 'refresh' ? visualRefreshRefactoredStyles['block-body-scroll'] : toolbarStyles['block-body-scroll'];
-  const unfocusableClassName = theme === 'refresh' ? visualRefreshRefactoredStyles.unfocusable : styles.unfocusable;
+      ? testUtilsStyles['mobile-bar']
+      : theme === 'refresh-toolbar'
+        ? toolbarSkeletonStyles['toolbar-container']
+        : mobileToolbarStyles['mobile-bar'];
+  const drawerBarClassName = theme.startsWith('refresh')
+    ? visualRefreshRefactoredStyles['drawers-mobile-triggers-container']
+    : drawersMobileStyles['drawers-container'];
+  const blockBodyScrollClassName = theme.startsWith('refresh')
+    ? visualRefreshRefactoredStyles['block-body-scroll']
+    : mobileToolbarStyles['block-body-scroll'];
+  const unfocusableClassName = theme.startsWith('refresh')
+    ? visualRefreshRefactoredStyles.unfocusable
+    : styles.unfocusable;
```

Comment:
These conditions do not separate `refresh` and `refresh-toolbar` classes. Any toolbar-mode test using these classes will not work correctly

Currently all of them are skipped. Is this your plan to update these conditions when enabling these tests?